src/main/java/Library/BookController.java

```diff
@@ -0,0 +1,36 @@
+package Library;
+import java.util.*;
+
+public class BookController {
+        private ArrayList<Book> bookList=new ArrayList<Book>();

```

Comment:
```suggestion
        private ArrayList<Book> bookList = new ArrayList<Book>();
```
esitlemelerden sonra bosluklara dikkat